.github/workflows/validations.yml

```
@@ -51,10 +59,12 @@ jobs:
           { "type": "module", "private": true }
           EOF
       - name: Add dependencies
+        env:
+          COREPACK_INTEGRITY_KEYS: 0
         run: |
-          yarn add express@${{matrix.express-version}} typescript@5.1 http-errors zod
-          yarn add eslint@9.0 typescript-eslint@8.0 vitest
-          yarn add express-zod-api@./dist.tgz
+          pnpm add express@${{matrix.express-version}} typescript@5.1 http-errors zod
+          pnpm add eslint@9.0 @typescript-eslint/parser@8.0 @typescript-eslint/utils@8.0 vitest
```

this can also be restored with `.npmrc` `public-hoist-pattern[]=@typescript-eslint/*`

```suggestion
          pnpm add eslint@9.0 typescript-eslint@8.0 vitest
```

---

reverted to yarn